Dockerfile-7.4-alpine

```diff
@@ -129,8 +107,17 @@ RUN apk update && \
     echo "extension=msgpack.so" > $CONF_PHPMODS/msgpack.ini && \
     pecl install memcached && \
     echo "extension=memcached.so" > $CONF_PHPMODS/memcached.ini && \
-    rm -rf /usr/share/php7 && \
+    # Disable xdebug by default \
+    sed -i 's/zend_extension\s\?=/;zend_extension =/' $CONF_PHPMODS/50_xdebug.ini && \
+    # Disable postgres by default \
+    sed -i 's/extension/;extension/' $CONF_PHPMODS/01_pdo_pgsql.ini && \
+    sed -i 's/extension/;extension/' $CONF_PHPMODS/00_pgsql.ini \
+    && \
+    /bin/bash -e /scripts/install-newrelic-alpine.sh && \
+    curl -sS https://getcomposer.org/installer | php && \
```

Comment:
getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md let's add this script to install composer instead of this. The checksum part is the piece that really interests me for security sake.

Reply:
done